app/presenters/menu/item.rb

```diff
@@ -40,10 +40,10 @@ def link_params
       params = case type.try(:to_sym)
                when :big_iframe then {:href => "/dashboard/iframe?id=#{id}"}
                when :new_window then {:href => href, :target => '_new'}
-               when :modal      then {'data-toggle' => 'modal', 'data-target' => href}
+               when :rxjs       then {:onclick => "sendDataWithRx({type: #{href.to_json}}); return false;", :href => 'javacript:void(0);'}
```

Comment:
this should *not* be using the `href` field to do magic

define you own

Reply:
Also, typo.. java**s**cript

Reply:
Well, I understand the problem, but creating a new field would require to create a new text field in the UI as well. That adds a lot of extra complexity, needs some design maybe even a UX review and IMO it's too risky for a backport.

Abusing the `href` field was a safer second option, we could also ignore the field and replace the *RxJS Action*  in the `type` dropdown with a more about modal specific option. Then it would be necessary to make the `href` field disabled, but this adds some extra complexity in the JS and I don't think I could do it without help.

Reply:
The mentioned UI with its default values:
![screenshot from 2018-10-18 10-41-19](https://user-images.githubusercontent.com/649130/47142075-64b6f380-d2c2-11e8-9c4f-d955e0680bd5.png)